pkgs/test_core/lib/src/runner/parse_metadata.dart

```diff
@@ -464,7 +464,7 @@ class _Parser {
     var list = expression as ListLiteral;
 
     // ignore: deprecated_member_use
```

Comment:
Can we remove the ignores now?

Reply:
That would still break build_web_compilers 1.x users

Reply:
> Can we remove the ignores now?

Done!